docs/skills/memory.md

```diff
@@ -14,6 +14,10 @@ Returns an object from the memory for the key provided.
 
 Stores the object provided for a specific key.
 
+### `delete(key)`
+
+Deletes the object provided the specific key.
+
 ### Example
```

Comment:
Let's move this example up, we can replace the `Persisting data` header with example one